README.md

```diff
@@ -138,6 +138,7 @@ our company and culture evolve, so should this document.
 - Take pride in how you build the software and your craftsmanship regardless of how the product does in the market. This doesn't mean gold-plating everything, often you should act pragmatically. Shipping value to users is the most important thing, but make sure to take the time to reflect on what you build, and do a post mortem and always make sure you're not
 repeating the mistakes of the past.
 - Ensure you challenge designs that you see problems with instead of implementing them without questioning.
+- Don't accept anything you find on the internet (StackOverflow, forums, etc) without understanding how it works and vetting it against the API documentation. It may be the accepted answer, but that does not mean it's correct.
```

Comment:
Ensure you understand and code found on the internet (StackOverflow, forums, etc) and vet against the API documentation. Often times the accepted answer is no longer correct.